### ISSUE TITLE
6.0.0-beta.0: Property '0' does not exist on type

### ISSUE THREAD

**Issue by marcjulian** [ngx-formly/ngx-formly#3314]
**Description**

**Minimal Reproduction**

I created a new Angular v13 project and added ngx-formly/core in version 6.0.0-beta.0.
Reproduction repo can be found here: https://github.com/marcjulian/formly-beta-6.

Do you have a stackblitz for Angular 13 and Formly 6.0.0-beta.0?

**Your Environment**

- Angular version: 13.3.10
- Formly version: 6.0.0-beta.0

**Additional context**

After starting 

```
ng s
✔ Browser application bundle generation complete.

Initial Chunk Files   | Names         |  Raw Size
vendor.js             | vendor        |   2.97 MB | 
polyfills.js          | polyfills     | 294.80 kB | 
styles.css, styles.js | styles        | 251.85 kB | 
main.js               | main          |  10.49 kB | 
runtime.js            | runtime       |   6.52 kB | 

                      | Initial Total |   3.52 MB

Build at: 2022-05-27T12:37:37.820Z - Hash: 72edc508cc6b48bf - Time: 3495ms

Error: node_modules/@ngx-formly/core/lib/services/formly.config.d.ts:37:52 - error TS2339: Property '0' does not exist on type '(string | Type<FieldWrapper<FormlyFieldConfig<FormlyFieldProps & { [additionalProperties: string]: any; }>>>)[] | undefined'.

37     getWrapper(name: FormlyFieldConfig['wrappers'][0]): WrapperOption;
                                                      ~
** Angular Live Development Server is listening on localhost:4200, open your browser on http://localhost:4200/ **
```

✖ Failed to compile.

**Comment by adora-tech** [comment]
I have the same problem, I confirm.
With next-9 was ok.

Thank you

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v6.0.0-beta.2](https://github.com/ngx-formly/ngx-formly/releases/tag/v6.0.0-beta.2) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!

**Comment by marcjulian** [comment]
@aitboudad thanks for the quick fix. It works fine with beta.2 👍
